cluster-up/cluster/kind-1.22-sriov/sriov-components/manifests/kustomization.yaml

```diff
@@ -8,10 +8,10 @@ resources:
 - sriovdp-config.yaml
 images:
   - name: ghcr.io/k8snetworkplumbingwg/sriov-network-device-plugin
-    newName: quay.io/kubevirtci/sriov-device-plugin
+    newName: ghcr.io/k8snetworkplumbingwg/sriov-network-device-plugin
```

Comment:
not sure I understand this commit

Reply:
As far as I know, on images are mirrored into kubevirtci registry in order to prevent pulling rate limiting issues (following dockerhub new policy), or pinning the image version.

@brianmcarey do you think we can use those images directly instead of mirroring them?

Reply:
the images that are pinned are only those we are mirroring manually + replace in the manifests
I dont think those are mirrored

on the other hand, ghcr.io is not docker.io, so it doesn't suffer rate limits (correct me if i am wrong)